tools/BuildAssets/BuildTasks/Microsoft.Azure.Sdk.Build.Tasks/BootstrapTasks/GetBuildTools.cs

```diff
@@ -129,10 +130,10 @@ public string RemoteRootDir
                         else
                         {
                             string url = remoteUri.ToString();
-                            if (url.ToLower().Contains("readme.md"))
+                            Regex regex = new Regex("readme.md", RegexOptions.IgnoreCase);
+                            if (regex.IsMatch(url))
```

Comment:
@dsgouda you will need to log an error using BuildToolsLogger.LogException/Error incase if the Match isn't successful.
Dump as much data as possible to know what went wrong, as to why we were not able to do a RegEx match.